.rultor.yml

```diff
@@ -0,0 +1,3 @@
+merge:
+  fast-forward: only
+  script: echo "Nothing to do."
```

Comment:
You don't have newline at the end of the file. Also `Fixes` -> `Closes` in the commit body. Fixes is for bugs